tools/scitokens-admin-oauthcerts

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+"""
+Create the oauth2 certs file from a given private or public key.  The resulting format
+should be something like:
+
+{
+    "keys": [
+        {
+            "alg": "RS256",
+            "e": "AQAB",
+            "kid": "ca78d4d8011b0442025c05885efba3f83764b436961d5cf09cec408ac6c675f2",
+            "kty": "RSA",
+            "n": "07llFSVsW8cXjy0kG2jHYm084QaSEZcrtw02fHbo30gxgdp6h-maIwNkj_xB-N29kUAe0McoJaL_P4P29rZfh_gh06f9fu60g_GPfVXBuKI61k1FfseaHLtwk2l20WOYHnx92v69UeylJEyNVVYNEhPUHEZdWWqzOjMUIC7XBnQ_GRiGu_9y7JFY-sIS28Iv36r7HKxv1k_i_B5LpOcZ0wUVxmk2WsflgSLg94iUAs8EU2ugI0ea8HG8hP6lPvcWdz4xIQlPYaovrLV_PgTOxVouye979BGIiWgkC7v0hYn5TV7xoIu_3ytm-MKFfas3MZ0cnGBWjlBqVNf-wNguSQ==",
+            "use": "sig"
+        }
+    ]
+}
+
+"""
+
+import sys
+import cryptography.utils
+from cryptography.hazmat.primitives import serialization, hashes
+from cryptography.hazmat.backends import default_backend
+import base64
+import binascii
+import json
+import argparse
+
+def string_from_long(data):
+    """
+    Create a base64 encoded string for an integer
+    """
+    return base64.urlsafe_b64encode(cryptography.utils.int_to_bytes(data)).decode('ascii')
+
+def bytes_from_long(data):
+    """
+    Create a base64 encoded bytes for an integer
+    """
+    return base64.urlsafe_b64encode(cryptography.utils.int_to_bytes(data))
+
+
+def add_args():
+    """
+    Generate the ArgumentParser object for the CLI.
+    """
+    parser = argparse.ArgumentParser(description='Create a public key for the oauth certs file')
```

Comment:
This is not really creating a public key, but rather formatting an existing key, isn't it?  How about

"Format a given public key (or create one from a private key) in the format needed for JWT headers."

Reply:
It's not for the jwt headers, it's for the OAuth2 Issuer.